CHANGELOG.md

```diff
@@ -28,6 +28,20 @@ For Firefox users, we provide limited scrollbar styling options through the foll
 - `--sb-thumb-bg-color`: Sets the background color of the scrollbar thumb.
 - `--sb-track-bg-color`: Sets the background color of the scrollbar track.
 
+#### Column editor options
+- `ColumnType`, `IgxColumn`
```

Comment:
There are at least two new features here, if not 3.
You have a general grid localiation of the input format used in editors (Edit, Filtering, etc UI), auto sync from the display format (pipeArgs.format) where applicable and then the new option to override the inferred format from either of the previous two.
Oh, and technically, those seem to be features baked into all date time editors and pickers too, right? (though how we describe that based on the current comments is TBD)

Reply:
Oh and I guess we should include the Query Builder in the components getting the new functionality

Reply:
@ddaribo I'd still describe the auto localized input format and infer from the display(pipe) format as a separate (first) point. That'll could even be more impactful for more people that the new option which is an override if the new functionality doesn't produce quite what you'd like :)